acceptance-tests/src/test/resources/uk/gov/di/test/acceptance/003_legal_and_policy_pages.feature

```diff
@@ -14,3 +14,22 @@ Feature: Legal and policy pages
     Then the user is taken to the terms and conditions page
     When the user clicks link "Privacy notice"
     Then the user is taken to the privacy notice page
+
+  Scenario: User accepts terms and conditions
+    Given the services are running
+    And a new user has valid credentials
+    And the new user clears cookies
+    When the new user visits the stub relying party
+    And the new user clicks "govuk-signin-button"
+    Then the new user is taken to the Identity Provider Login Page
+    When the new user selects sign in
+    When the new user enters their t&c email address
+    When the new user enters their t&c password
+    When the new user enters the six digit security code from their phone
```

Comment:
Should these be `And` instead of `When`?

Reply:
Yes! I changed 26 and 27 to And, but the When on line 28 I kept as such as it's used across many scenarios and would mean almost duplicating methods. Thanks :)